mantidproject/mantid
# Rename tabs of QENS Fitting interface

### Description of work
After renaming the Data Analysis tab to QENS Fitting, we think the names of the tabs are a bit redundant (MSD Fit, I(Q,t) Fit, ConvFit...) as it is clear from the interface name that its purpose is to fit different types of data.  The new names for the tabs are: MSD, I(Q,t), Convolution and Function(Q). In summary: 
- Changed tabs names. 
- Changed related classes names. 
- Updated docs and dev-docs references. 

More information can be found in #36853. 

Additionally, I have added a tooltip on the tabs, which will raise a helpful info message of what each tab does whenever the mouse is hovered on that tab. 

To be noted: The `I(Q,t) Fit` is renamed as `I(Q, t)`  on the GUI, but the underlying classes and files preserve the name starting with `IqtFit...` as there is a nother `Iqt..` class on the Data Processor interface that interferes with it, and it will raise confusion if both classes share the same name (for example there would be two `IqtModel` files). 


Fixes #36853 . 



### To test:
- Build dev-docs. 
- Follow `Inelastic QENS Fitting Manual Testing` instructions on the built docs, making sure the instructions contain the updated tab names. 
- Make sure the manual tests run without problem, and the tabs on `QENS Fitting` interface have the new names. 
- Build docs, check the `Interfaces`->`QENS Fitting` interface page and check everything looks updated with the new names and properly formatted. 



*Added release notes*


---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.
